src/main/java/com/poortorich/email/request/EmailVerificationRequest.java

```diff
@@ -1,17 +1,18 @@
 package com.poortorich.email.request;
 
-import jakarta.validation.constraints.Email;
+import com.poortorich.email.constants.EmailResponseMessage;
+import com.poortorich.email.validator.annotations.Email;
 import jakarta.validation.constraints.NotBlank;
 import lombok.AllArgsConstructor;
 import lombok.Getter;
 
 @Getter
 @AllArgsConstructor
 public class EmailVerificationRequest {
-    @NotBlank(message = "invalid_email")
-    @Email(message = "invalid_email")
+    @NotBlank(message = EmailResponseMessage.INVALID_EMAIL)
+    @Email
     private String email;
```

Comment:
```suggestion

    @NotBlank(message = EmailResponseMessage.INVALID_EMAIL)
    @Email
    private String email;
```

클래스 선언문이랑 변수 사이에 공백 넣어주세요 👀